src/components/label/label.svelte

```diff
@@ -23,20 +22,22 @@
   export let color: Color = 'gray'
 
   $: background =
-    color === 'CTA'
-      ? `var(--leo-gradient-hero)`
-      : mode === 'loud'
-      ? `var(--leo-color-${color}-40)`
-      : `var(--leo-color-${color}-10)`
+    mode === 'loud' ? `var(--leo-color-${color}-50)` : 'transparent'
+
   $: text =
-    color === 'CTA'
-      ? `var(--leo-color-white)`
-      : mode === 'loud'
+    mode === 'loud'
       ? `var(--leo-color-${color}-10)`
       : `var(--leo-color-${color}-50)`
+
+  $: border = mode === 'loud' ? `transparent` : `var(--leo-color-${color}-50)`
```

Comment:
Just wondering why not use a `.loud` class and vary these in the css?

Reply:
I thought it would add an extra layer of indirection, for not much gain. It would mean we'd need a class for each color, which then handles `.loud` separately:

```svelte

<div class:red={color === "red"} class:blue={color === "blue"} .... class:loud={mode === 'loud'}/>

<style>
.red {
  color: var(--leo-color-red-50);
  background: var(--leo-color-red-10);
  border: 1px solid var(--leo-color-red-50);

  &.loud {
    color: var(--leo-color-red-10);
    background: var(--leo-color-red-50);
    border-color: transparent; 
  }
}

/* repeat for each color */
</style>
```

or do you mean something more like?

```svelte
<script>
const lighter = `var(--leo-color-${color}-10`
const darker = `var(--leo-color-${color}-50`
</script>

<div class="label" style:--darker=${darker} style--lighter=${lighter} />

<style>
.label {
    color: var(--darker);
    background: transparent;
    border: 1px solid var(--darker);
}

.label.loud {
    color: var(--lighter);
    background: var(--darker);
    border-color: transparent;
}
</style>
```

Reply:
To be honest, I didn't notice the `color` embedded expression! So it's all good

I guess you could do

```css
--border-color: var(--leo-color-{color}-50);

.loud {
  --border-color: transparent;
}
```

Or does that binding / string interpolation not work in svelte css, only html attributes?

Reply:
Exactly, yeah. Nothing in the <style> tag supports interpolation :cry: